translations/Base_Norwegian.lang

```diff
@@ -0,0 +1,133 @@
+mysqlPane.credentials.enterAction: "**Klikk** for å **skrive inn** MySQL Legitimasjon"
+mysqlPane.credentials.password: "Passord: **%**"
+mysqlPane.credentials.title: "Legitimasjon"
+mysqlPane.credentials.username: "Brukernavn: **%**"
+mysqlPane.database: "database"
+mysqlPane.disable.action: "Klikk for å skru av MySQL"
+mysqlPane.disable.desc: "Du vil bli spurt om å restarte for å bruke de nye endringene"
+mysqlPane.disable.title: "Deaktiver"
+mysqlPane.encryption.disableAction: "**Klikk** for å **skru av** kryptering"
+mysqlPane.encryption.enableAction: "**Klikk** for å **skru på** kryptering"
+mysqlPane.encryption.ssl: "SSL: **%**"
+mysqlPane.encryption.title: "SSL"
+mysqlPane.everythingSaved.desc: "Det er ingen endringer å lagre"
+mysqlPane.everythingSaved.title: "Alt er lagret"
+mysqlPane.hostname: "vertsnavn"
+mysqlPane.loading.desc: "Vent mens programtillegget kjører alle tilkoblingstester"
+mysqlPane.loading.title: "Tester tilkoblinger"
+mysqlPane.name: "MySQL Database"
+mysqlPane.password: "passord"
+mysqlPane.port: "port"
+mysqlPane.save.action: "Klikk for å **Lagre** endringene"
+mysqlPane.save.title: "Lagre"
+mysqlPane.setup.action: "Klikk for å skru av MySQL"
+mysqlPane.setup.desc: "Dette trenger en eksternell MYSQL database"
+mysqlPane.setup.title: "Oppsett MySQL"
+mysqlPane.test.action: "Klikk for å teste **endringene**"
+mysqlPane.test.database: "Database: **%**"
+mysqlPane.test.hostname: "Vertsnavn: **%**"
+mysqlPane.test.maximumPoolSize: "Maksimal pool størrelse: **%**"
+mysqlPane.test.minimumIdle: "Minimum Tomgang: **%**"
+mysqlPane.test.password: "Passord: **%**"
+mysqlPane.test.port: "Port: **%**"
+mysqlPane.test.ssl: "SSL: **%**"
+mysqlPane.test.title: "Prøv endringene"
+mysqlPane.test.username: "Brukernavn: **%**"
+mysqlPane.username: "brukernavn"
+mysqlSetup.actionBar: "Standarinstillinger: **%**"
+mysqlSetup.subtitle: "Skriv in **%** i Chatten"
+mysqlSetup.title: "MySQL Setup"
+none: "none"
+pageableView.basicSearch.action: "Klikk for å søke"
+pageableView.basicSearch.title: "Søk"
+pageableView.next.action: "Klikk for å gå til neste side"
+pageableView.next.title: "Neste"
+pageableView.previous.action: "Klikk for å gå til den forje siden"
+pageableView.previous.title: "Forje"
+pageableView.search.closeAction: "klikk for å **avbryte** søket"
+pageableView.search.subtitle: "Skriv inn et søkssord"
+pageableView.search.title: "Søk"
+restartView.button.action: "Klikk for å restarte Serveren"
+restartView.button.title: "Restart"
+restartView.restarting: "Restarter..."
+restartView.title: "Restart Serveren"
+second: "sekund"
+seconds: "sekunder"
+settings: "instillinger"
+settingsView.select.action: "Klikk for å se instillinger"
+settingsView.select.indicator: "Denne siden er **valgt**"
+settingsView.title: "Instillinger > %"
+timePickerView.addDay: "Legg til 1 Dag"
+timePickerView.addHour: "Legg til 1 Time"
+timePickerView.addMinute: "Legg til 1 minutt"
+timePickerView.addSecond: "Add 1 sekund"
```

Comment:
Everything after this line (the line included) still needs translation. Did you forget it?

Reply:
Oh, sorry, didn’t notice it. I will finish it as fast as i can!

Reply:
@DAmNRelentless finished it!